ynr/apps/resultsbot/management/commands/store_modgov_urls.py

```diff
@@ -0,0 +1,26 @@
+import csv
+import os
+
+from django.core.management.base import BaseCommand
+
+import resultsbot
+from elections.models import Election
+
+
+class Command(BaseCommand):
```

Comment:
I think we need to consider how to delete these URLs if we find them not to exist / they go down / whatever. 

Do you see this as a field that we should persist in the long run on the model, or could we delete all values before inserting them form the CSV?

Reply:
Yeah that's a good point - I am thinking very short term really. So I think deleting them before reimporting them would work, as it would allow us to remove any bad ones from the CSV file before we re-run this management command?

Reply:
Longer term refactor we'd probably want to handle it in the actual running of `resultsbot_import_from_mg_csv` somewhere - so if we know a URL is bad/has no data, we could update the related Election model.

Reply:
Yeah, I agree!